### DISCUSSION TITLE
List ignored files for integration with other tools

### DISCUSSION THREAD

**Discussion by omasanori** [jj-vcs/jj#1396] [Ideas]
Something like `git ls-files --ignored` or `hg status --ignored` will be useful for tools with "ignore files ignored by VCS" option. For instance, [REUSE tool, a license scanner, use the mentioned commands to exclude ignored files in Git or Mercurial working directories](https://github.com/fsfe/reuse-tool/blob/main/src/reuse/vcs.py).

**Comment by martinvonz** [comment]
Since files that are not ignored are automatically added (unless we address #323), maybe the "is ignored" check in REUSE could be implemented as "not in `jj files`". Do you think that would work?

**Reply by omasanori** [reply]
Thanks! I will try. One more question: is there an equivalent command to `hg root`? `jj workspace root` seems a bit different concept.

**Reply by martinvonz** [reply]
That should be the same concept. I think they both print the path to the current workspace (or "share").

**Reply by omasanori** [reply]
I see. Thanks a lot!
